hw/ip/aes/aes.core

```diff
@@ -24,8 +24,58 @@ filesets:
       - rtl/aes.sv
     file_type: systemVerilogSource
 
+  files_verilator_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+    files:
+      - lint/aes.vlt
+    file_type: vlt
+
+  files_ascentlint_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+      # tool setup
+      - lowrisc:lint:ascentlint
+    files:
+      - lint/aes.waiver
+    file_type: waiver
+
+
+parameters:
+  SYNTHESIS:
+    datatype: bool
+    paramtype: vlogdefine
+
+
 targets:
   default: &default_target
     filesets:
+      # Note on Verilator waivers:
+      # You *must* include the waiver file first, otherwise only global waivers
+      # are applied, but not file-specific waivers.
+      - tool_verilator  ? (files_verilator_waiver)
+      - tool_ascentlint ? (files_ascentlint_waiver)
       - files_rtl
     toplevel: aes
+
+  lint:
```

Comment:
There should be only one "lint" target, it can support both tools by passing the appropriate `--tool=xxx` option to the fusesoc command line.

Reply:
Good point, that should make the core files a bit "leaner".

Reply:
done